cv32e40s/regress/cv32e40s_full.yaml

```diff
@@ -409,8 +415,7 @@ tests:
 #TODO:MT Comment back in when Zc is enabled in the core
 #  zc_test:
 #    description: Zc directed test
-#    builds: [ uvmt_cv32e40s, uvmt_cv32e40s_pma_1]
+#    builds: [uvmt_cv32e40s, uvmt_cv32e40s_pma_1]
```

Comment:
@silabs-mateilga do you know if there is a specific reason to use pma config "1" and not one of the others?